### DISCUSSION TITLE
CSS is included twice when using `nuxt generate`?

### DISCUSSION THREAD

**Discussion by vaaski** [nuxt/nuxt#28523] [Questions] [answered]
Hey, quick question. While trying to debug a production build of my site, I've noticed that some CSS gets included twice. Once as a `<style>` element and again as a `<link rel="stylesheet">`. While trying to research this I've found #16849 and #14953, but the issue still persists in the latest stable build.

### To reproduce:
- Use the latest nuxt starter [nuxt.new/s/v3](https://nuxt.new/s/v3)
- Run `npm run generate`
- Inspect the generated `index.html` with `code .output/public/index.html`
- The `<style>` on L5 and contents of the CSS file imported on L6 seem identical

### For comparison

<details>
<summary>The inline <code><style></code></summary>

<img width="665" alt="image" src="https://github.com/user-attachments/assets/b76684db-b7ea-4be1-8092-a6346e9afff6">

</details>

<details>
<summary>The contents of the CSS file</summary>

<img width="662" alt="image" src="https://github.com/user-attachments/assets/64014b30-3836-44b5-8263-1374e00a3338">

</details>

**Comment by xjccc** [comment, marked as answer]
```
  features: {
    inlineStyles: false
  },
```
